src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/HtmlErrorTest.cs

```diff
@@ -10,51 +10,51 @@ public class HtmlErrorTest : MarkupParserTestBase
         [Fact]
         public void AllowsInvalidTagNamesAsLongAsParserCanIdentifyEndTag()
         {
-            ParseBlockTest("<1-foo+bar>foo</1-foo+bar>");
+            ParseDocumentTest("@{<1-foo+bar>foo</1-foo+bar>}");
         }
 
         [Fact]
         public void ThrowsErrorIfStartTextTagContainsTextAfterName()
         {
-            ParseBlockTest("<text foo bar></text>");
+            ParseDocumentTest("@{<text foo bar></text>}");
         }
 
         [Fact]
         public void ThrowsErrorIfEndTextTagContainsTextAfterName()
         {
-            ParseBlockTest("<text></text foo bar>");
+            ParseDocumentTest("@{<text></text foo bar>}");
         }
 
         [Fact]
         public void ThrowsExceptionIfBlockDoesNotStartWithTag()
         {
-            ParseBlockTest("foo bar <baz>", expectedParseLength: 0);
+            ParseDocumentTest("@{foo bar <baz>}");
         }
 
         [Fact]
         public void StartingWithEndTagErrorsThenOutputsMarkupSegmentAndEndsBlock()
         {
             // ParseBlockStartingWithEndTagProducesRazorErrorThenOutputsMarkupSegmentAndEndsBlock
-            ParseBlockTest("</foo> bar baz", expectedParseLength: 7);
+            ParseDocumentTest("@{</foo> bar baz}");
         }
 
         [Fact]
         public void WithUnclosedTopLevelTagThrowsOnOutermostUnclosedTag()
         {
             // ParseBlockWithUnclosedTopLevelTagThrowsMissingEndTagParserExceptionOnOutermostUnclosedTag
-            ParseBlockTest("<p><foo></bar>");
+            ParseDocumentTest("@{<p><foo></bar>}");
         }
 
         [Fact]
         public void WithUnclosedTagAtEOFThrowsMissingEndTagException()
         {
-            ParseBlockTest("<foo>blah blah blah blah blah");
+            ParseDocumentTest("@{<foo>blah blah blah blah blah}");
```

Comment:
For this and the test below, I think the closing `}` can be removed. Looks like it is testing EOF scenarios.

Reply:
Hmm, actually for all of these shouldn't I just nuke all of the "Throws" xyz tests?

Reply:
These tests should probably be renamed. `Throws` is the wrong word. It just adds a parser error.
But don't nuke these tests expect the one. I assume these errors still exist (in *.diag.txt).

Reply:
I fixed the name of this test but the diff isn't showing the fix.